style.css

```diff
@@ -448,3 +489,463 @@ footer .contact-para {
   margin-top: 18px;
   font-size: 18px;
 }
+
+@media screen and (min-width: 992px) and (max-width: 1100px) {
```

Comment:
- [ ] Kindly remove the `(max-width: 1100px)` from your media query. With max-width, the browser will not load the desktop version for the screens whose sizes are greater than 1100px. In our project, we defined the minimum breakpoint for our desktop version to be 992px.